server/graphql/common/expenses.ts

```diff
@@ -165,7 +165,7 @@ const isCollectiveAdmin = async (req: express.Request, expense: Expense): Promis
   return req.remoteUser.isAdminOfCollective(expense.collective);
 };
 
-const isHostAdmin = async (req: express.Request, expense: Expense): Promise<boolean> => {
+export const isHostAdmin = async (req: express.Request, expense: Expense): Promise<boolean> => {
```

Comment:
If used outside of this file, we should call it `isExpenseHostAdmin`. Otherwise another developer will likely use is as `isHostAdmin(req, collective)` at some point.